proto/sedachain/batching/v1/batching.proto

```diff
@@ -44,19 +49,45 @@ message Signature {
   uint32 scheme = 1;
   // signature is the signature of the batch.
   string signature = 2;
-  // public_key is the public key corresponding to the signature that 
+  // public_key is the public key corresponding to the signature that
   // should have been registered in x/pubkey
   string public_key = 3;
   // merkle_proof is the proof of inclusion of the signer in the validator
-  // tree. 
+  // tree.
   string merkle_proof = 4;
 }
 
-// Module parameters which can be changed through governance.
+// Params is a list of parameters which can be changed through governance.
 message Params {
   option (gogoproto.equal) = true;
 
   // validator_set_trim_percent is the percentage of the validator
   // set to store in the validator merkle tree in the batch.
   uint32 validator_set_trim_percent = 1;
 }
+
+// DataResult represents the result of a resolved data request.
+message DataResult {
+  // id is the Keccack-256 hash of the data result.
+  string id = 1;
+  // dr_id is the data request identifier.
+  string dr_id = 2;
+  // version is a semantic version string.
+  string version = 3;
+  // block_height is the height at which the data request was tallied.
+  uint64 block_height = 4;
+  // exit_code is the exit code of the tally wasm binary execution.
+  uint32 exit_code = 5;
```

Comment:
Yes it should be `uint8`, but protobuf doesn't have that type :/ 
seda-wasm-vm should be fixed to return a `u8` from the rust side and `uint8` from the go side, right?

Reply:
Yes, we can make an issue for that, or maybe I'll just do it in the logging pr.

Reply:
Oh... wasmer returns an `i32` as an exit code I see...

Reply:
I don't see in the wasi codebase where it could be negative but I asked a question to the `wasi` team. Let's see what they say and maybe we should be keeping it as an `i32`.

Reply:
Thank you! I am just casting it to uint8 on the module side for now. Franklin mentioned that `process::exit` can only use 8 bits.

Reply:
I'm waiting on a response here https://github.com/wasmerio/wasmer/issues/5131 for this.

Reply:
Actually, we discussed this, instead on the tally VM we should take this exit code from WASM and put that exit code in the string and reserve a number for WASMER exit codes.